### PR TITLE
chore: add static check if one struct implements ConfigInterface

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

**Comment by hezhizhen** [comment]
@caoxianfei1 PTAL

File: pkg/chunkserver/config.go [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
-import "strconv"
+import (
+	"strconv"
+
+	"github.com/opencurve/curve-operator/pkg/config"
+)
 
-// chunkserverConfig for a single chunkserver
 // chunkserverConfig implements config.ConfigInterface
+var _ config.ConfigInterface = &chunkserverConfig{}
```
Comment (caoxianfei1) [inline pkg/chunkserver/config.go:10]: init func() maybe better?

Comment (caoxianfei1) [reply pkg/chunkserver/config.go:10]: IMO

Comment (hezhizhen) [reply pkg/chunkserver/config.go:10]: Not necessary. It's only a check and not used anywhere.

Comment (hezhizhen) [reply pkg/chunkserver/config.go:10]: And that's why I use _ as the variable name.

Comment (caoxianfei1) [reply pkg/chunkserver/config.go:10]: ok, make successed? because ci system is not added.

Comment (hezhizhen) [reply pkg/chunkserver/config.go:10]: If the struct doesn't implement the interface, go build will fail, so does `make run`.

We can add CI later.

Comment (caoxianfei1) [reply pkg/chunkserver/config.go:10]: If you are willing to together so much the better^^

**Comment by hezhizhen** [comment]
@caoxianfei1 Ready to merge it?

**Comment by caoxianfei1** [comment]
> @caoxianfei1 Ready to merge it?

I will merge it a little later.

**Review by caoxianfei1 — APPROVED** [review]

**Pull request merged**